grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -525,4 +525,15 @@ class OrderService {
 
         return orderItems.findAll { !it.isCompletelyFulfilled() }
     }
+
+    def canOrderItemBeEdited(OrderItem orderItem, String userId) {
```

Comment:
Sorry to be nit-picky here but let's go with isOrderItemEditable. The only reason I use the canDoSomething or hasSomething is when there is no isSomethingTrue that makes sense. Here, that's not the case.

Reply:
Actually, maybe you're right. Since this is more about whether the user can do something and less about whether the item can have an action taken against it ...

... so disregard.

Reply:
Ok